src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import optionStyles from './option.scss';
+
+const SELECTED_ICON = 'check';
+
+export default class Option extends React.PureComponent {
+    props: {
+        selected: boolean,
+        disabled: boolean,
+        focus: boolean,
+        value: string,
+        children?: string,
+        onClick?: (s: string) => void,
+    };
+
+    static defaultProps = {
+        disabled: false,
+        selected: false,
+        focus: false,
+        value: '',
+    };
+
+    item: HTMLElement;
+    button: HTMLButtonElement;
+
+    componentDidMount() {
+        if (this.props.focus) {
```

Comment:
You are right, only one select can be opened at a time and the problem I described is really impossible. However, the element which receives the focus can also be the second option even if the second option is not selected (when the first one should receive the focus but is disabled). Moreover, when using these options in a multiple select it's impossible to couple the selected and focused state.